Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -514,6 +512,51 @@ def get_attributes_command(client: Client, args: Dict[str, str], params: Dict[st
     )
 
 
+def update_candidate(last_run: dict, last_run_timestamp: str, last_run_value: str,
+                     latest_indicator_timestamp: str, latest_indicator_value: str):
```

Comment:
```suggestion
def update_candidate(last_run: dict, last_run_timestamp: int, last_run_value: str,
                     latest_indicator_timestamp: int, latest_indicator_value: str):
```

Reply:
why int? the value is retrieved from the last run as a str.